src/Illuminate/Validation/Validator.php

```diff
@@ -1026,7 +1026,7 @@ protected function callExtension($rule, $parameters)
     {
         $callback = $this->extensions[$rule];
 
-        if ($callback instanceof Closure) {
+        if ($callback instanceof Closure || is_array($callback)) {
```

Comment:
I believe you can change this line into something like this: `if (is_callable($callback)) {`. And it will achieve the same result.